e2e/pages/app/select/custom-heading/select-custom-heading.testpage.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+    <div class="col-xs-6">
+        <ux-select
+            id="dropdown"
+            [options]="options"
+            [(recentOptions)]="recentOptions"
+            [placeholder]="placeholder"
+            [optionsHeadingTemplate]="demoHeader"
+            [recentOptionsHeadingTemplate]="demoRecentHeaderOptions">
+        </ux-select>
+    </div>
+    <div class="col-xs-6">
+        <div>
+            <ux-checkbox id="checkbox1" (valueChange)="recentOptions = $event ? [] : undefined">
+                recentOptions
+            </ux-checkbox>
+        </div>
+    </div>
+</div>
+
+<ng-template #demoHeader>
+    <div id="Nodes">Nodes</div>
```

Comment:
```suggestion
    <div id="options-header">Nodes</div>
```
Try to use meaningful IDs/variable names.